### DISCUSSION TITLE
OWON PCT504 NOT supported?

### DISCUSSION THREAD

**Discussion by cpomodoro** [Koenkk/zigbee2mqtt#15662] [Devices]
Hi, I've got an OWON PCT504-E fancoil thermostat. It is included in the supported devices list, but actually, after pairing with my Home Assistant (sonoff zigbee 3.0 usb dongle), it reported as "NOT supported" and it cannot be controlled.
Is there any advice for fixing this problem?
Thank you.

**Comment by cpomodoro** [comment]
Hi, I submit the question again. The OWON PCT504-E is included among supported devices, but it is reported as "NOT supported" after pairing. Can anybody confim? Thank you.

**Comment by luismipr** [comment]
Same problem.
I bought an PCT504-E and it sais not supported... why?
https://www.zigbee2mqtt.io/devices/PCT504.html

This is the termostat.

Any solution?

Thx.

**Comment by morb3at** [comment]
Same problem. 

please update if there is any solution

**Comment by luismipr** [comment]
I have created a new devices.js with:

`module.exports = [
    {
        zigbeeModel: ['PCT504-E'],
        model: 'PCT504-E',
        vendor: 'OWON',`

It works perfectly, except humidity and occupied.

This is a extract from db:

`{"id":6,"type":"Router","ieeeAddr":"0x3c6a2cfffed152ec","nwkAddr":31579,"manufId":4412,"manufName":"OWON Technology Inc.","powerSource":"DC Source","modelId":"PCT504-E","epList":[1,2],"endpoints":{"1":{"profId":260,"epId":1,"devId":769,"inClusterList":[0,3,4,5,513,514,516,1029],"outClusterList":[10],"clusters":{"genBasic":{"attributes":{}},"hvacThermostat":{"attributes":{"runningState":4,"localTemp":1750,"occupiedCoolingSetpoint":2500,"occupiedHeatingSetpoint":1750,"ctrlSeqeOfOper":4,"systemMode":4,"runningMode":4,"tempSetpointHold":1,"tempSetpointHoldDuration":65535,"programingOperMode":0}},"hvacUserInterfaceCfg":{"attributes":{"tempDisplayMode":0,"keypadLockout":0}},"hvacFanCtrl":{"attributes":{"fanMode":1}}},"binds":[{"cluster":0,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":3,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":4,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":513,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":516,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":514,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":1026,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1},{"cluster":1030,"type":"endpoint","deviceIeeeAddress":"0x00124b002a2e90e5","endpointID":1}],"configuredReportings":[{"cluster":514,"attrId":0,"minRepIntval":0,"maxRepIntval":3600,"repChange":0},{"cluster":513,"attrId":18,"minRepIntval":0,"maxRepIntval":3600,"repChange":10}],"meta":{}},"2":{"profId":260,"epId":2,"devId":263,"inClusterList":[0,3,1030],"outClusterList":[],"clusters":{"genBasic":{"attributes":{"stackVersion":0}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":1,"stackVersion":0,"hwVersion":4,"dateCode":"20200513","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1680382735261,"defaultSendRequestWhen":"immediate"}`

Then... how can overryde device.

I need to create a new file or i can do something in configuration.yaml 

Thx.
